autoshop/core/urls.py

```diff
@@ -9,8 +9,7 @@
 urlpatterns = [
     url(r'^$', views.index, name='index'),
     url(r'^register/$', views.register, name='register'),
-    url(r'^failure/$', views.failure, name='failure'),
-    url(r'^success/$', views.success, name='success'),
+    url(r'^dashboard/$', views.dashboard, name='dashboard'),
```

Comment:
remove this page as our requirement for now is to redirect user to homepage, i.e. `index`.